JAICore/jaicore-ml/src/jaicore/ml/core/dataset/attribute/categorical/ICategoricalAttributeType.java

```diff
@@ -0,0 +1,19 @@
+package jaicore.ml.core.dataset.attribute.categorical;
+
+import java.util.Set;
+
+import jaicore.ml.core.dataset.attribute.IAttributeType;
+
+/**
+ * Interface for categorical attribute types.
+ *
+ * @author wever
+ */
+public interface ICategoricalAttributeType extends IAttributeType<String> {

```

Comment:
What is the purpose of this interface? Isn't this somehow duplicate with the underlying class?

Reply:
This is on purpose, as there might be other implementations in the future. However, these should be unified having at least the same interface.

Reply:
Ah, yes - that makes sense.